helm/idol-library/templates/aciserver/_deployment.tpl

```diff
@@ -88,6 +53,9 @@ spec:
       {{- range $component.additionalVolumes }}
       - {{ . | toYaml | nindent 8 }}
       {{- end }}
+      {{- if $component.podSecurityContext.enabled }}
+      securityContext: {{- omit $component.podSecurityContext "enabled" | toYaml | nindent 8 }}
```

Comment:
Sure you've tested the templating here, but slightly surprised there's no newline here.

Reply:
I think all the conversion back and forth to yaml in the merge code deals with all the formatting